src/js/components/visualization/Visualization.jsx

```diff
@@ -93,6 +93,68 @@ class Visualization extends Component {
       typeGroup
     }
   }
+  
+  renderError (error) {
+    const {visualization} = this.props
+    const {name} = visualization
+    let text = error.message
+    
+    try {
+      text = JSON.parse(error.message)
+    } catch (err) {}
+    
+    return (
+      <div style={style.container}>
+        <VisualizationToolbar
+          title={name}
+          onClose={this.onClose}
+        />
+        <div style={{flex: 1, position: 'relative'}}>
+          <span
+            style={{
+              color: 'red',
+              left: '50%',
+              position: 'absolute',
+              top: '50%',
+              transform: 'translate(-50%, -50%)'
+            }}
+          >
+            {(() => {
+              if (typeof text === 'string') {
```

Comment:
I had talked to Patrick about this. He suggested allowing a String or Object. I guess we could have just an object and if a string is needed, then set the 'label' field.